taxonomies/labels.txt

```diff
@@ -19712,3 +19712,11 @@ fr:Sans résidus de pesticides, sans résidu de pesticide, zéro résidus de pes
 # obligation of means: cultivated without pesticides (but can be a specific year, or after flowers bloom)
 en:Pesticide-free, without pesticide, no pesticide
 fr:Sans pesticides, cultivé sans pesticides
+
+fr:Prix Juste Producteur
+country:en:Belgium
+auth_name:fr:Services Opérationnels du Collège des Producteurs
+auth_address:fr:Avenue Comte de Smet de Nayer 14, 5000 Namur, Belgium
+auth_url:fr:https://collegedesproducteurs.be/
```

Comment:
@alexgarel, is there a way to add a url to the label (not just the authority url) : https://prixjuste.be/

Reply:
You can use **web:fr:** as prefix. Is there a wikidata and/or wikipedia entry? If yes you can add those as well.